Fixed a few typos in the doc (#2339)
cyberbotics/webots: docs/guide/using-matlab.md

Note that this is similar to calling "matlab" from a terminal (or *Command Prompt* on Windows), therefore, if MATLAB can be started from a terminal then it can also be started from Webots.

On Windows, the MATLAB installer will normally add MATLAB's bin directories to your *Path* environment variable, so usually, Webots will be able to locate MATLAB after a standard installation.

> However, in case it does not work, please make sure that your *Path* environment variable contains this directory (or something slightly different, according to your MATLAB version): `C:\Program Files\MATLAB\R2020b\bin`.

Reviewer: We should probably use:
```suggestion
However, in case it does not work, please make sure that your *Path* environment variable contains this directory (or something slightly different, according to your MATLAB version): `C:\Program Files\MATLAB\{{ webots.version.major }}\bin`.
```

Author: I am hesitating about this as it turns out MATLAB versions often lag behind Webots versions: MATLAB R2020a was released in March 2020 and Webots R2020a was released in December 2019...

Otherwise you can create the link at anytime afterwards with this shell command (please change according to your actual MATLAB installation directory and version):

```sh

> $ sudo ln -s /usr/local/MATLAB/R2020b/bin/matlab /usr/local/bin/matlab

Reviewer: ```suggestion
$ sudo ln -s /usr/local/MATLAB/{{ webots.version.major }}/bin/matlab /usr/local/bin/matlab
```

```

Similarly, on macOS, if Webots is unable to find the "matlab" startup script then you should add a symlink in "/usr/bin":

```sh

> $ sudo ln -s /Applications/MATLAB_R2020b.app/bin/matlab /usr/local/bin/matlab

Reviewer: ```suggestion
$ sudo ln -s /Applications/MATLAB_{{ webots.version.major }}.app/bin/matlab /usr/local/bin/matlab
```
